engine/security/src/test/scala/pl/touk/nussknacker/ui/security/http/RecordingSttpBackend.scala

```diff
@@ -0,0 +1,44 @@
+package pl.touk.nussknacker.ui.security.http
+
+import java.util.concurrent.atomic.AtomicReference
+
+import sttp.client.monad.MonadError
+import sttp.client.ws.WebSocketResponse
+import sttp.client.{Request, Response, SttpBackend}
+
+import scala.concurrent.Future
+import scala.util.{Failure, Success, Try}
+
+class RecordingSttpBackend(delegate: SttpBackend[Future, Nothing, Nothing]) extends SttpBackend[Future, Nothing, Nothing] {
```

Comment:
Let's add comment that it comes from https://github.com/softwaremill/sttp/blob/master/core/src/main/scala/sttp/client3/testing/RecordingSttpBackend.scala and that it should be removed when we migrat to sttp3? Or if we added some methods - just a notice that it's based on sttp version